### DISCUSSION TITLE
THANK YOU!!!

### DISCUSSION THREAD

**Discussion by rmp5s** [audiamus/AaxAudioConverter#130] [General]
Such a MASSIVE problem-solver...perfect app.  Getting my auth code was a _little_ tricky, but I found [some website](https://audible-converter.ml/) that could provide it if you toss it an AAX file.  No problem.

Just wanted to say thanks and, if you've got a PayPal or something, I'd really like to pitch you a few bucks!

Thanks again!!

**Comment by audiamus** [comment]
Thank you for you kind feedback and your offer. Please don't feel offended, but this is really just a hobby project, it does not need money,

Regarding the activation bytes, my software, both AAX Audible Converter and its new companion [Book Lib Connect](https://github.com/audiamus/BookLibConnect), only look for official sources of the codes. This worked well with legacy Audible Manager and with the now also discontinued Audible Win 10 app. Since both are retired or about to be, obtaining the codes has become more difficult. With Book Lib Connect, however, you can now directly download your books from the server, including their more complex per-book code, and Book Lib Connect will decrypt them. AAX Audible Converter can still be used afterwards to tailor the books to your needs.

**Comment by rmp5s** [comment]
`Thank you for you kind feedback and your offer. Please don't feel offended, but this is really just a hobby project, it does not need money,`

Hey, no problem.  I really appreciate your contribution and always try to support those that do things like this.  It's greatly appreciated!

`Regarding the activation bytes, my software, both AAX Audible Converter and its new companion [Book Lib Connect](https://github.com/audiamus/BookLibConnect), only look for official sources of the codes. This worked well with legacy Audible Manager and with the now also discontinued Audible Win 10 app. Since both are retired or about to be, obtaining the codes has become more difficult. With Book Lib Connect, however, you can now directly download your books from the server, including their more complex per-book code, and Book Lib Connect will decrypt them. AAX Audible Converter can still be used afterwards to tailor the books to your needs.`

Oh, nice!  I didn't know about the other program.  Good to hear there is an easier way to do it.  I got it figured out (I now have all my audiobooks backed up locally and organized neatly!), and it wasn't _too_ difficult, but it did take a couple extra steps.  Sounds like you've gotten that sorted, though.  Hopefully, at least the next person that comes along as I did sees this post and is saved the extra work.

Anyway...thanks again!

**Reply by Masala-ms** [reply]
Just quickly would like to chime in with the praise here. Was looking for a paypal address or the like here on github and the manual and only saw the exceptionally implemented documentation instead.

In case you're wondering what I'm using it for (because in my car, I'm absolutely using over-engineered Android Auto): here's a wall of text :)
BLC/AaxAC is my rescue for my small kids' (privacy-friendly) mp3 player+speaker. In addition to bluetooth streaming (from my cell phone), it lets them select one of several playlists stored on an SD card - and will only play mp3/m4a (or wav, really). Before I found your software -- and after many fails with other free or shareware software claiming to convert Audible and/or Spotify but was long outdated or downloaded similar YouTube audio only -- I streamed my Audible/Spotify audios from my phone to my PC via analogue cable (_de: Klinke_), recorded with Audacity, cut with m3u, and finally created file names with mp3tag. Boy, was that a hassle, and I didn't do it often. My kids will love you dearly as they will be getting new audiobooks now much more often than they used to.

So, Thank you!!
